frontend/src/container/MetricsApplication/Tabs/util.ts

```diff
@@ -31,7 +31,8 @@ export function onViewTracePopupClick({
 }: OnViewTracePopupClickProps): VoidFunction {
 	return (): void => {
 		const currentTime = timestamp;
-		const tPlusOne = timestamp + 60 * 1000;
+
+		const tPlusOne = timestamp + 50;
```

Comment:
50?

Reply:
If this is a custom logic, we can add a comment explaining it as part of code.

Reply:
50 was overridden can you re-check.... @YounixM